src/process_utility.c

```diff
@@ -906,6 +906,8 @@ process_vacuum(ProcessUtilityArgs *args)
 	Hypertable *ht;
 	List *vacuum_rels = NIL;
 	bool is_vacuumcmd;
+	/* save original VacuumRelation list */
+	List *stmt_rels = list_copy(stmt->rels);
```

Comment:
```suggestion
	List *stmt_rels = stmt->rels;
```
Do we really need to copy the list?? Below we're assigning `stmt->rels` to another memory area, no? Specifically in this line:
```
stmt->rels = list_concat(ctx.chunk_rels, vacuum_rels);
```

Reply:
Ok, fixed.